### DISCUSSION TITLE
Dialogs Prompt Dialog

### DISCUSSION THREAD

**Discussion by SnowYeti** [NativeScript/NativeScript#9650] [Q&A]
On Android, the Softkeyboard hides the OK/Cancel Buttons when trying to get user input via the Prompt Dialog.
I would be happy about any hint as a workaround.
Thank you!!!
Anja

**Comment by JacobFJ** [comment]
Can you share snippet or screenshots of your concern?

**Comment by SnowYeti** [comment]
Yes, thanks for asking!

please find it attached.

Ok/cancel Buttons are hidden behind the Keyboard

⁣BlueMail for Android herunterladen ​

**Comment by JacobFJ** [comment]
I see you can set your Android Manifest to ```android:windowSoftInputMode="adjustResize"```

So your layout can adjust when the keyboard was open.

**Comment by SnowYeti** [comment]
Hi Hakobi,
thank you very much but unfortunately it did not help. I rebuild the App and send screenshot of the manifest.
Is there any possibility to change the keyboard for the dialog? Set done instead of new line or catch the key press?


Again thank you very much for your help!

Kind regards
Anja

⁣BlueMail for Android herunterladen ​

**Comment by farfromrefug** [comment]
the dialogs does not seem to follow the main activity soft input mode as explained here https://github.com/NativeScript/NativeScript/blob/master/packages/core/ui/core/view/index.android.ts#L218. 
you either need to find a way to change it for dialogs in the android style or modify N to add the same kind of option as I just linked (which is for modal views) to dialogs prompt

**Comment by farfromrefug** [comment]
it is explained here https://stackoverflow.com/questions/25215248/how-to-make-fragment-honor-soft-input-mode-of-activity
